src/utils/legacy-settings.js

```diff
@@ -50,9 +50,11 @@ const LegacySettingIds = {
   HIDE_SUBSCRIPTION_NOTICES: 'hideSubscriptionNotices',
   HIDE_COMMUNITY_HIGHLIGHTS: 'hideCommunityHighlights',
   HIDE_PRIME_PROMOTIONS: 'hidePrimePromotions',
-  HIDE_FEATURED_CHANNELS: 'hideFeaturedChannels',
   AUTO_EXPAND_CHANNELS: 'autoExpandChannels',
   HIDE_OFFLINE_FOLLOWED_CHANNELS: 'hideOfflineFollowedChannels',
+  HIDE_RECENTLY_WATCHED_CHANNELS: 'hideRecentlyWatchedChannels',
```

Comment:
these are legacy setting keys, and should not be created or destroyed. the only change to this file might be setting new settings based on legacy settings

Reply:
Thank you for the explanation, I modified this file according on this.